packages/by-name/contrast/package.nix

```diff
@@ -27,9 +27,9 @@ let
     subPackages = [ "e2e/genpolicy" "e2e/getdents" "e2e/openssl" "e2e/servicemesh" "e2e/release" ];
   };
 
-  launchDigest = builtins.readFile "${runtime-class-files}/launch-digest.hex";
+  launchDigest = builtins.readFile "${runtime.runtime-class-files}/launch-digest.hex";
```

Comment:
Does this mean we need to build dedicated for an environment? The idea was to allow multiple reference value sets (and multiple defaults, afaict) in a single binary.

Reply:
The node-installer is platform-agnostic, but the container image we ship for it (including the runtime files) would get unreasonably large if we'd include all platforms.